projects/packages/my-jetpack/_inc/data/evaluation-recommendations/use-evaluation-recommendations.ts

```diff
@@ -27,9 +30,18 @@ const useEvaluationRecommendations = () => {
 		'recommendedModules',
 		getInitialRecommendedModules()
 	);
+	const [ isFirstRun, setIsFirstRun ] = useValueStore( 'isFirstRun', getInitialIsFirstRun() );
 
 	const unownedRecommendedModules = useMemo( () => {
-		const { ownedProducts = [] } = getMyJetpackWindowInitialState( 'lifecycleStats' );
+		// TODO: Maybe remove this ternary condition
+		// This check is for local development & testing purposes because the monorepo local dev
+		// environment unrealistically returns ALL the products/plugins as owned products, resulting
+		// in zero(0) unownedRecommendedModules.
```

Comment:
I ran into this too when testing. Since we are forcing the Welcome banner to be visible during development, it has this weird issue. However if there really are that many owned products that the user has, they won't see the welcome banner 😄